### PR TITLE
2-Create-Account (resolves #2)

### LINKED ISSUE #2: Create "Create an Account" Page
### Description
Create a basic "create an account" page based on [Figma](https://www.figma.com/file/MNRQAXLXzBqCR3aRZmXg0T/Habitat-for-Humanity). Include error handling for input fields. If an error occurs, an alert pops up describing the error. Store all input fields in a state variable so it can be formatted as a JSON object later
_* Note: I would just name this component CreateAccount, rather than CreateAnAccount_

### Setup
1. Navigate to `/frontend/src/components/`
2. Create a folder with the [PascalCase](https://techterms.com/definition/pascalcase) naming convention (name based on the component you're working on)
3. Inside the folder, create a `.tsx` file with the same name as the folder
4. Be sure to update the `App.js` file with your new component, or else you wont see any changes you make. This file is located in `/frontend/src/`

### Success Metrics Checklist
- [x] Looks exactly like the Figma
- [x] Responsive screen handling
- [x] Error pops up for invalid/empty input field
- [x] Store submitted input into state variable

Finally, on your pull request, please request a code review from me (@ObjectJosh)

Screen Shot 2022-01-07 at 11 04 08 PM

### PR DESCRIPTION
**Pull request by scallait**
-Added CSS styling to match Figma
-Responsive page handling
-Basic form error checking
-Converts form data to JSON format
-Converts phone number to number type

### CODE REVIEW

File: frontend/src/components/CreateAccountPage/CreateAccountPage.css [hunk 1]
```diff
@@ -0,0 +1,219 @@
+@import url('https://fonts.googleapis.com/css?family=Rubik&display=swap');
+
+#createAccountBox {
+    width: 675px;
+    height: 760px;
```
Comment (ObjectJosh) [inline frontend/src/components/CreateAccountPage/CreateAccountPage.css:5]: Create account box is taking the full height of the screen for larger screen sizes:
Screen Shot 2022-01-10 at 10 39 09 PM
vs. Figma:
Screen Shot 2022-01-10 at 10 39 36 PM

**Review by ObjectJosh — CHANGES_REQUESTED** [review]
Great job Sebastien! I appreciate you commenting the function descriptions as well! Just a fix with the box size in relation to the screen. Other than that, looks good!

File: frontend/src/components/CreateAccountPage/CreateAccountPage.css (hunk 1 above)
Comment (scallait) [reply frontend/src/components/CreateAccountPage/CreateAccountPage.css:5]: Adjustment done. As for the create account text, would you prefer the bolded version or the non-bolded version?

**Nonbolded:**
image

**Bolded:**
image

Comment (ObjectJosh) [reply frontend/src/components/CreateAccountPage/CreateAccountPage.css]: As a developer, we need aim for consistency whenever possible. By the looks of it (comparing to other screens), probably the bold. What do you think?

Comment (scallait) [reply frontend/src/components/CreateAccountPage/CreateAccountPage.css]: I came to the same conclusion. Thank you for your suggestions.

**Review by ObjectJosh — APPROVED** [review]
Looks great! _(note for future reference: the bug may or may not still be there, will check with other members)_

**Pull request merged**